### PR TITLE
Nepnep

### PR DESCRIPTION
**Pull request by GameFuzzy**
Nepnep nepnep nepnep

### CODE REVIEW

**Review by TheNetsky — DISMISSED** [review]
Looks decent for something by Fuzzy's standards

**Review by nar1n — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "extensions-official",
+  "name": "extensions-nepnep",
   "version": "1.0.0",
```
Comment (FramboisePi) [inline package.json:3]: Would you mind adding a `repositoryName` name field?
Something like:
```
"repositoryName": "Extensions NepNep",
```

**Review by FramboisePi — CHANGES_REQUESTED** [review]
Hum are you sure about the base branch? I think it should be `NepNep`

**Review by FramboisePi — APPROVED** [review]
Trying to approve this pull request

**Review by FramboisePi — APPROVED** [review]
Trying to approve this pull request

**Review by TheNetsky — APPROVED** [review]
If this doesn't work, Paper didn't fix shit

**Pull request merged**